src/client/components/FooterComponent/FooterComponent.styles.css

```diff
@@ -0,0 +1,13 @@
+.footer-component {
+    background: #FFFFFF;
+    width: 375px;
```

Comment:
Is it not possible to use a more flexible layout? Usually large hardcoded values for padding, margin, width and height means its not responsive

Reply:
What kind of responsive values do you recommend me to use instead of px?

Reply:
https://github.com/HackYourFuture-CPH/fp-class12/pull/46

Reply:
I changed it to 100% so it can take container's set width. I also updated height to em but should I also consider it giving it a % value?

Reply:
Ususally you try to avoid setting a height to elements that are not part of the outer 'frame' so to speak. Best to avoid it all together. Is the height property needed?

Reply:
I am just checking without height property and it is still the same result. So it's not needed :)